### PR TITLE
Removed hard symfony/proxy-manager-bridge dependecy

### PR DESCRIPTION
**Pull request by caciobanu**
Related to https://github.com/SimpleBus/SymfonyBridge/issues/66

### CODE REVIEW

**Comment by caciobanu** [comment]
Code coverage is lower but I think that this is normal. I'll think of another way to test with and without the symfony/proxy-manager-bridge in the same test suite to get a more accurate code coverage.

**Comment by ruudk** [comment]
@cmodijk Codecov is such a party pooper 🤡, I wish it would be gone completely :pray:

**Comment by ruudk** [comment]
When we merge this, we have to tag this a new major release, right?. We also need to add it to the CHANGELOG so that people know that this changed.

**Comment by caciobanu** [comment]
Yes, I think that we should consider it a major version.

Should we consider keeping a change log and an upgrade file like Symfony does ?

File: src/DoctrineOrmBridgeBundle.php [hunk 1]
```diff
@@ -28,11 +28,26 @@ public function build(ContainerBuilder $container)
     {
         $this->checkRequirements(array('SimpleBusCommandBusBundle', 'SimpleBusEventBusBundle'), $container);
 
+        $this->checkProxyManagerBrideIsPresent();
```
Comment (mvrhov) [inline src/DoctrineOrmBridgeBundle.php]: `checkProxyManagerBridgeIsPresent`

File: src/DoctrineOrmBridgeBundle.php [hunk 2]
```diff
@@ -28,11 +28,26 @@ public function build(ContainerBuilder $container)
...
+
         $compilerPass = new AddMiddlewareTags(
             'simple_bus.doctrine_orm_bridge.wraps_next_command_in_transaction',
             ['command'],
             100
         );
         CompilerPassUtil::prependBeforeOptimizationPass($container, $compilerPass);
     }
+
+    private function checkProxyManagerBrideIsPresent()
```
Comment (mvrhov) [inline src/DoctrineOrmBridgeBundle.php]: `checkProxyManagerBridgeIsPresent`

**Comment by mvrhov** [comment]
So can we have this merged after the typos are fixed.

File: src/DoctrineOrmBridgeBundle.php (hunk 1 above)
Comment (caciobanu) [reply src/DoctrineOrmBridgeBundle.php]: Good catch!

File: src/DoctrineOrmBridgeBundle.php (hunk 2 above)
Comment (caciobanu) [reply src/DoctrineOrmBridgeBundle.php]: Good catch!

**Pull request merged**